src/main/java/htsjdk/samtools/util/BlockCompressedInputStream.java

```diff
@@ -57,10 +57,12 @@
     public final static String INCORRECT_HEADER_SIZE_MSG = "Incorrect header size for file: ";
     public final static String UNEXPECTED_BLOCK_LENGTH_MSG = "Unexpected compressed block length: ";
     public final static String PREMATURE_END_MSG = "Premature end of file: ";
-    public final static String CANNOT_SEEK_STREAM_MSG = "Cannot seek on stream based file ";
```

Comment:
It's a clearer name but you're right, it's breaks backward compatibility and is not really need. I will revert the name.